docs/_generate/index.html

```diff
@@ -139,7 +139,15 @@
       <h2><a href='#sources' title='link to sources'>Sources</a></h2>
       <p>
         Sources supply data to be shown on the map. The type of source is specified by the <code>"type"</code> property,
-        and must be one of <var><%= source_types.join(', ') %></var>. Tiled sources (vector and raster) must specify
+        and must be one of <var><%= source_types.join(', ') %></var>. Sources
+        supply data but don't make it appear on the map - they don't contain
```

Comment:
This sentence is a little repetitive to the first sentence

 > Sources supply data to be shown on the map.

> Sources supply data but don't make it appear on the map...